src/skvhttpclient/src/skvhttp/dto/SKVRecord.h

```diff
@@ -229,7 +229,8 @@ class SKVRecordBuilder {
             }
         }
 
-        _writer.write(field);
```

Comment:
I don't think this is right - the value should be written as-is. We read it as T on the reverse path, not optional<T>